reana-ui/src/components/WorkflowList.js

```diff
@@ -82,11 +82,11 @@ export default class WorkflowList extends Component {
    * Gets data from the specified API
    */
   getData() {
-    fetch(URL + "access_token=" + TOKEN)
+    const { column, direction, token } = this.state;
+
+    fetch(URL + "access_token=" + token)
```

Comment:
here you can also use template literals, instead of concatenating the vars. Check this one https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals